lib/aws-sdk-rails.rb

```diff
@@ -1,10 +1,13 @@
 # frozen_string_literal: true
 
 require_relative 'aws/rails/mailer'
+require_relative 'aws/rails/notifications_instrument_plugin'
```

Comment:
you're going to need to rebase this against my change once merged, or vice versa, up to you.